packages/amplify-graphql-auth-transformer/src/graphql-auth-transformer.ts

```diff
@@ -794,6 +799,97 @@ Static group authorization should perform as expected.`,
     );
   };
 
+  addAutoGeneratedFields = (
```

Comment:
these seem like util functions can we move these into the utils section?

Reply:
updated